src/translations/default.json

```diff
@@ -764,7 +764,7 @@
   "modules.reserve-overview.components.ReserveInformation.totalBorrowed": "Total Borrowed",
   "modules.reserve-overview.components.ReserveInformation.usedAsCollateral": "Used as collateral",
   "modules.reserve-overview.components.ReserveInformation.utilisationRate": "Utilisation rate",
-  "modules.reserve-overview.components.ReserveInformation.variableBorrowing": "Variable borrowing",
```

Comment:
I don't think we need to change the key `variableBorrowing` here. It's still possible sometime in future, we may enable both variable and stable borrowing.